MediaBrowser.Providers/Plugins/Tmdb/TmdbUtils.cs

```diff
@@ -188,5 +192,66 @@ public static string BuildParentalRating(string countryCode, string ratingValue)
 
             return newRating.Replace("DE-", "FSK-", StringComparison.OrdinalIgnoreCase);
         }
+
+        /// <summary>
+        /// Removes invalid characters for file names from the search result name for better comparsison.
+        /// </summary>
+        /// <param name="searchResultName">The name from the search result.</param>
+        /// <returns>The input with invalid file name characters removed.</returns>
+        public static string? RemoveInvalidFileCharacters(string? searchResultName)
+        {
+            if (searchResultName == null)
+            {
+                return null;
+            }
+
+            string formatted = _fileSystemInvalid.Replace(searchResultName, string.Empty);
+            return formatted;
+        }
+
+        /// <summary>
+        /// Identifies the nearest match to the search criteria. Addresses issues with search results from TMDB.
+        /// </summary>
+        /// <param name="name">The cleaned up name searched for.</param>
+        /// <param name="year">The year searched for.</param>
+        /// <param name="searchResults">The search results from a Movie or Tv search.</param>
+        /// <returns>The nearest match to the search criteria.</returns>
+        public static SearchMovieTvBase? GetBestMatch(string name, int year, IEnumerable<SearchMovieTvBase> searchResults)
```

Comment:
```suggestion
        public static SearchMovieTvBase? GetBestMatch(string name, int year, IReadOnlyList<SearchMovieTvBase> searchResults)
```

using `IEnumerable` here raises warnings